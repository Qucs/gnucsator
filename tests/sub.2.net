# Qucs 0.0.19  /tmp/neur.sch

.Def:TESTSUB _net0 _net1 RES="1" PARAM2="2"
resistor:a _net0 _net1 r="RES"
.Def:End

.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
Sub:T1 _net1 0 Type="TESTSUB" RES="2"
Idc:V1 _net1 0 I="5"
