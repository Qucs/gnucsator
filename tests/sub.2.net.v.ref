--- sub.2.net.v.out	2018-09-19 02:01:31.035945868 +0100
+++ sub.2.net.v.ref	2018-09-19 01:49:18.763975897 +0100
@@ -1,4 +1,5 @@
 
+vsource #(.dc(0.)) p (.p(gnd),.n(0));
 module TESTSUB (_net0,_net1);
 parameter PARAM2={2};
 parameter RES={1};
