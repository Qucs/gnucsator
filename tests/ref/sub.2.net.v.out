
module main ();
//# Qucs 0.0.19  /tmp/neur.sch 
module TESTSUB (_net0,_net1);
parameter PARAM2={2};
parameter RES={1};
resistor #(.r(RES)) a (.p(_net0),.n(_net1));
endmodule // TESTSUB

TESTSUB #(.RES(2),.Type(TESTSUB)) T1 (._net0(_net1),._net1(0));
Idc #(.I(5)) V1 (.1(_net1),.2(0));
endmodule // main

